### DISCUSSION TITLE
is it possible to use reshade with mpv?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14424] [General]
Hi, I would like to know if reshade is compatible with mpv. There was nothing in the manual about that.

Thank you :)

**Comment by ghost** [comment]
https://reshade.me/forum/general-discussion/5596-image-editor-as-reshade-host#35296

**Reply by geextahslex** [reply]
I don't understand.

**Reply by ghost** [reply]
You asked if it's possible to use reshade with mpv. The answer is yes.

**Reply by geextahslex** [reply]
Oh okay, thank you.
